blackbox-multi-scope/src/main/java/org/multi/scope/CrossCutScope.java

```diff
@@ -2,9 +2,9 @@
 
 import io.avaje.inject.InjectModule;
 import jakarta.inject.Scope;
-import org.multi.modb.BeanInModB;
 
 @Scope
-@InjectModule(requires = {ModAScope.class, ModBScope.class}, strictWiring = true)
-public @interface CrossCutScope {
-}
+@InjectModule(
+    requires = {ModAScope.class, ModBScope.class},
+    provides = CrossCutScope.class)
```

Comment:
To me the `provides` should **_not_** be needed ultimately. To me we've needed to add them here ... but ultimately that wouldn't be desirable.

Reply:
Would auto-providing the scope annotation suffice?